lms/static/js/student_account/components/StudentAccountDeletion.jsx

```diff
@@ -73,7 +81,15 @@ export class StudentAccountDeletion extends React.Component {
           className="account-settings-header-subtitle"
           dangerouslySetInnerHTML={{ __html: loseAccessText }}
         />
-
+        <p className="account-settings-header-subtitle-warning">
+          <strong>{ gettext('Warning: Account deletion is permanent. ') }</strong>
+          {gettext('Please read the above carefully before proceeding. This is an irreversible action, and ')}
+          <strong>{ gettext('you will no longer be able to use the same email on edX.') }</strong>
```

Comment:
LInes 86 and 87 will need to be interpolated in the same way ```changeAcctInfoText``` was for the translations to work as expected.

Reply:
Thanks! I've fixed it in a way that I hope will work.